custom_components/alexa_media/switch.py

```
@@ -51,7 +52,7 @@ async def async_setup_platform(hass, config, add_devices_callback, discovery_inf
     if config:
         account = config.get(CONF_EMAIL)
     if account is None and discovery_info:
-        account = discovery_info.get("config", {}).get(CONF_EMAIL)
+        account = dictor(discovery_info, f"config.{CONF_EMAIL.replace(".", "\\.")}")
```

Pylint (4.0.4)

[error] Parsing failed: 'f-string: unmatched '(' (alexa_media.switch, line 55)'

(E0001)